resources/js/features/events/components/AchievementSet/AchievementsListItem/AchievementsListItem.tsx

```diff
@@ -0,0 +1,173 @@
+import { useTranslation } from 'react-i18next';
+
+import { BaseProgress } from '@/common/components/+vendor/BaseProgress';
+import { AchievementAvatar } from '@/common/components/AchievementAvatar';
+import { GameTitle } from '@/common/components/GameTitle';
+import { cn } from '@/common/utils/cn';
+import { formatNumber } from '@/common/utils/l10n/formatNumber';
+import { formatPercentage } from '@/common/utils/l10n/formatPercentage';
+
+import { AchievementDateMeta } from '../AchievementDateMeta';
+
+interface AchievementsListItemProps {
+  achievement: App.Platform.Data.Achievement;
+  index: number;
+  isLargeList: boolean;
+  playersTotal: number;
+
+  /**
+   * Wherever possible, map stuff onto `achievement`.
+   * The less smart this component is, the easier it'll be to maintain long-term.
+   * Only pick unique fields from this prop, such as `activeUntil`.
+   */
+  eventAchievement?: App.Platform.Data.EventAchievement;
+}
+
+export const AchievementsListItem: FC<AchievementsListItemProps> = ({
+  achievement,
+  index,
+  isLargeList,
+  eventAchievement,
+  playersTotal,
+}) => {
+  const { t } = useTranslation();
+
+  const { title, description, game } = achievement;
+
+  const unlockHardcorePercentage = achievement.unlockHardcorePercentage ?? 0;
+  const unlocksHardcoreTotal = achievement.unlocksHardcoreTotal ?? 0;
+  const unlocksTotal = achievement.unlocksTotal ?? 0;
+
+  return (
+    <motion.li
+      className="flex w-full gap-x-3 px-2 py-3 odd:bg-[rgba(50,50,50,0.4)] light:odd:bg-neutral-200 md:py-1"
+      initial={{ opacity: 0, y: 10 }}
+      animate={{ opacity: 1, y: 0 }}
+      exit={{ opacity: 0, y: 10 }}
+      transition={{
+        duration: 0.12,
+        delay: isLargeList
+          ? Math.min(index * 0.008, 0.15) // Cap at 150ms for large lists
+          : Math.min(index * 0.015, 0.2), // Cap at 200ms for small lists
+      }}
+    >
+      <div className="flex flex-col gap-y-1">
+        <AchievementAvatar
+          {...achievement}
+          showLabel={false}
+          hasTooltip={false}
+          size={64}
+          displayLockedStatus="auto"
+        />
+      </div>
+
+      <div className="mt-1 grid w-full gap-x-5 gap-y-1.5 leading-4 md:grid-cols-6">
+        {/* Title and description area */}
+        <div className="md:col-span-4">
+          <div className="mb-0.5 flex justify-between gap-x-2">
+            {/* Title */}
+            <div className="-mt-2 mb-0.5 md:mt-0">
+              <a href={route('achievement.show', { achievement })} className="font-medium">
+                {title}
+                {game?.title ? ' ' : null}
+              </a>
+
+              {game?.title ? (
+                <>
+                  <span>{t('from')}</span>{' '}
+                  <a href={route('achievement.show', { achievement })}>
+                    <GameTitle title={game.title} isWordWrappingEnabled={true} />{' '}
+                    {`(${game.system!.nameShort})`}
+                  </a>
```

Comment:
Can we get a game card tooltip on this?

Reply:
Added in latest. Also fixed a bug where these game titles were not linking to the correct page.